ccmlib/cmds/cluster_cmds.py

```diff
@@ -693,8 +693,6 @@ def get_parser(self):
         parser = self._get_default_parser(usage, self.description())
         parser.add_option('--no-hh', '--no-hinted-handoff', action="store_false",
                           dest="hinted_handoff", default=True, help="Disable hinted handoff")
-        parser.add_option('--batch-cl', '--batch-commit-log', action="store_true",
```

Comment:
I'd way rather reimplement it on top of the new implementation. I don't know why we wouldn't do that. I have too many users to possibly exhaustively ask them all, and the drivers team hates when I remove things, and make them put it back [for good reason].